vector/src/main/res/values/strings.xml

```diff
@@ -322,6 +322,14 @@
     <string name="start_chatting">Start Chatting</string>
     <string name="spaces">Spaces</string>
 
+    <!-- Time units -->
+    <plurals name="time_unit_hour_short">
```

Comment:
Maybe add a comment for translator that if a short version exists in their language, it has to be used. For English there is no short version, so it can confuse the translators.

Reply:
Done see [e9d9319](https://github.com/vector-im/element-android/pull/5989/commits/e9d93194f15f58c3ce10e132b6fc244de4c8f5bb)